src/hint_processor/builtin_hint_processor/secp/ec_utils.rs

```diff
@@ -136,6 +138,30 @@ pub fn compute_slope(
     exec_scopes.insert_value("slope", value);
     Ok(())
 }
+pub fn compute_slope_local_secp_p(
```

Comment:
This one could be unified with `compute_slope` in a similar way to [`div_mod_n_packed`](https://github.com/lambdaclass/cairo-rs/blob/f470b0e51f74a8eed57c1579a0beb6e9146317de/src/hint_processor/builtin_hint_processor/secp/signature.rs#L52-L70). That way we keep the `import` part separate from the hint implementation.